src/components/StudyView/StudyView.tsx

```diff
@@ -150,6 +213,34 @@ const StudyView: React.FC<StudyViewProps> = ({ submissionId }) => {
             </IonItem>
           </IonList>
 
+          <SectionHeader>Templates</SectionHeader>
+          <IonList className="ion-padding-bottom">
+            {templateVisibleSlots.map((item) => (
+              <IonItem
+                key={item.template}
+                onClick={() => setModalTemplateVisibleSlots(item)}
+              >
+                <IonLabel>
+                  <h3>{item.templateDisplay}</h3>
+                  <p>
+                    {item.visibleSlots === undefined ? (
+                      "Not customized"
+                    ) : (
+                      <>
+                        <Pluralize
+                          count={item.visibleSlots.length}
+                          singular={"field"}
+                          showCount
+                        />{" "}
+                        chosen
```

Comment:
I don't know exactly what this message looks like in context. Another word that came to mind is "visible" (as an alternative to "chosen").

Reply:
I went with chosen here because even though the words visible and visibility are all over the code, they don't really appear much in the final interface.

The text in the modal itself actually starts with the imperative "Select..." so maybe "selected" is even better here.